ddtrace/tracer/textmap_test.go

```diff
@@ -2875,3 +2875,61 @@ func TestExtractBaggagePropagatorMalformedHeader(t *testing.T) {
 		assert.Len(t, got, 0)
 	})
 }
+
+func TestExtractOnlyBaggageGeneratesNonZeroTraceID(t *testing.T) {
+	t.Setenv("DD_TRACE_PROPAGATION_STYLE", "baggage")
+	headers := TextMapCarrier(map[string]string{
+		"baggage": "foo=bar,baz=qux",
+	})
+
+	tracer, err := newTracer()
+	assert.NoError(t, err)
+	defer tracer.Stop()
+
+	ctx, err := tracer.Extract(headers)
+	assert.NoError(t, err)
+
+	zeroTrace := traceIDFrom64Bits(0)
+	assert.NotEqual(t, zeroTrace, ctx.traceID)
```

Comment:
I agree the context should have a trace ID, I think I'm missing something about the baggage propagation, as I see that we test it completely isolated but I'm not sure if users will use only the baggage to propagate traces.